tools/wake-format/formatter.h

```diff
@@ -35,34 +35,34 @@
 // #define NL_STR "⏎\n"
 
 struct ctx_t {
-  size_t width = 0;
   size_t nest_level = 0;
+  wcl::doc_state state = wcl::doc_state::identity();
 
-  ctx_t nest() {
+  ctx_t nest() const {
     ctx_t copy = *this;
     copy.nest_level++;
     return copy;
   }
 
-  ctx_t sub(const wcl::doc_builder& builder) {
+  ctx_t operator+(const wcl::doc_builder& builder) const {
```

Comment:
nit: I think we should go back to `sub` 

But this 100% should be `const`...sorry I missed that before!